From 04ad89953422df46bfce1f6d45b676ce2e612d1f Mon Sep 17 00:00:00 2001
From: Lin Jianhua <linjh@rock-chips.com>
Date: Wed, 26 Dec 2018 15:00:01 +0800
Subject: [PATCH] arm64: dts: rockchip: rk3308: modify default pinctrl of tsadc
 from tsadc_otp_out to tsadc_otp_gpio

Some HW connect to peripheral reset circuit,
but uboot first select set pinctrl mode to default,
it result system reboot.

Change-Id: I79453dd4839e846138130e6488aca9c0f419fb38
Signed-off-by: Lin Jianhua <linjh@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3308.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3308.dtsi b/arch/arm64/boot/dts/rockchip/rk3308.dtsi
index a5e83d147748..554a76b398d2 100644
--- a/arch/arm64/boot/dts/rockchip/rk3308.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3308.dtsi
@@ -844,7 +844,7 @@
 		reset-names = "tsadc-apb";
 		pinctrl-names = "init", "default", "sleep";
 		pinctrl-0 = <&tsadc_otp_gpio>;
-		pinctrl-1 = <&tsadc_otp_out>;
+		pinctrl-1 = <&tsadc_otp_gpio>;
 		pinctrl-2 = <&tsadc_otp_gpio>;
 		#thermal-sensor-cells = <1>;
 		rockchip,hw-tshut-temp = <120000>;
-- 
2.35.3

